pipeline/src/test_util.rs

```diff
@@ -207,7 +205,8 @@ pub fn test_halo2_with_backend_variant(
     let is_nightly_test = env::var("IS_NIGHTLY_TEST")
         .map(|v| v == "true")
         .unwrap_or(false);
-    if is_nightly_test {
+
+    if is_nightly_test && should_generate_proofs() {
```

Comment:
I'm actually not sure we set this for the nightly run....

Can you check?

Reply:
Looks like it: https://github.com/powdr-labs/powdr/blob/e58ddac313963f7c4a57f2ad4760b1f54eba6791/.github/workflows/nightly-tests.yml#L9

Reply:
I mean the generate_proofs variable

Reply:
Oh, no, that one we don't :D Let me fix it.